README.md

```diff
@@ -21,17 +21,14 @@ export APIGEN_SPECIFICATION_VERSION=<version_string>
 export APIGEN_MAINTENANCE_REPORT_PATH=<maintenance_report_dir>
 ```
 
-### Adding new BlackDuckResponse's
-All views will be BlackDuckComponent by default. To change BlackDuckComponent's to BlackDuckResponse's, ClassCategories must be modified.
-
-### Adding new LinkResponse's
-To add a new LinkResponse to a view, LinkResponseDefinitions needs to be modified to add the new link to the view.
-
 ### Generate Files
 Once the environment variables are set, run the following command in the project's root directory: 
 ```bash
 ./gradlew bootRun
 ```
+##### Notes:
+* If a generated directory already exists at APIGEN_GENERATED_DIRECTORY_PATH, it should be deleted prior to a new run of the generator.
+* For API specification versions <2020.12.0, the specification file at endpoints/api/projects/projectId/versions/projectVersionId/comparison/GET/<latest media type>/response-specification.json contains two "items" fields.  One is an array of objects, one is of type "Array" but has no fields.  Delete the one that has no fields prior to running the generator.
```

Comment:
Interesting, so you delete it from the spec before running the generator. This issue does show up in multiple places though, so do you want to mention the other? Or make a more general note?

Reply:
By "multiple places" do you mean places other than VersionBomComponentDiffView?